src/index.js

```diff
@@ -1,30 +1,50 @@
 import './style.css';
 
-const displayTable = document.querySelector('.display-table');
+const inputName = document.querySelector('.input-name');
+const inputScore = document.querySelector('.input-score');
+const submitBtn = document.querySelector('.submit-btn');
+const displayLists = document.getElementById('display-lists');
+const refreshBtn = document.querySelector('.refresh-btn');
 
-const displayScore = [
-  {
-    Name: 'Mire',
-    Score: 100,
-  },
-  {
-    Name: 'Biruk',
-    Score: 30,
-  },
-  {
-    Name: 'Bini',
-    Score: 50,
-  },
-  {
-    Name: 'Sami',
-    Score: 90,
-  },
-  {
-    Name: 'Yoni',
-    Score: 70,
-  },
-];
+// Unique identifier of the created game
+const gameId = 'j3pbXgKmDrCFgfpG7CfU';
 
-displayTable.innerHTML = displayScore.map((e) => `
-        <p class="display-table-list">${e.Name}: ${e.Score}</p>
-    `).join('');
\ No newline at end of file
+async function submitScore(userName, userScore) {
+  const response = await fetch(`https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/${gameId}/scores`, {
+    method: 'POST',
+    body: JSON.stringify({
+      user: userName,
+      score: userScore,
+    }),
+    headers: {
+      'Content-type': 'application/json; charset=UTF-8',
+    },
+  });
+  const gameResult = await response.json();
+  return gameResult;
+}
+
+async function fetchDataFromAPI() {
+  const response = await fetch(`https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/${gameId}/scores`);
+  const getResult = await response.json();
```

Comment:
- [ ] Ensure to use of arrow functions instead of the `function` keyword as this is a project requirement to use ES6 syntax.

Reply:
Resolved